### PR TITLE
Customer confirm email and customer change email

### PR DESCRIPTION
**Pull request by gillesbourgeat**
Hello,

Add the ability to ask customers to change and confirm their email.

configuration variable customer_confirm_email.
0 ==> no confirm
1 ==> confirm

configuration variable customer_change_email.
0 ==> no confirm
1 ==> confirm

For customer_confirm_email :
- Default config value 1 for new thelia install.
- Default config value 0 for thelia update, to keep compatibility with older templates.

For customer_change_email :
- Default config value 0

Cheers,

### CODE REVIEW

File: setup/insert.sql [hunk 1]
```diff
@@ -147,7 +152,9 @@ INSERT INTO `config_i18n` (`id`, `locale`, `title`, `description`, `chapo`, `pos
 (33, 'fr_FR', 'Utiliser un cookie persistant pour memoriser le panier du client', NULL, NULL, NULL),
 (34, 'fr_FR', 'Nom du cookie de stockage du panier', NULL, NULL, NULL),
 (35, 'fr_FR', 'Durée de vie du cookie du panier dans le navigateur du client, en secondes', NULL, NULL, NULL),
-(36, 'fr_FR', 'Durée de vie du cookie de la session dans le navigateur du client, en secondes', NULL, NULL, NULL)
+(36, 'fr_FR', 'Durée de vie du cookie de la session dans le navigateur du client, en secondes', NULL, NULL, NULL),
+(37, 'fr_FR', 'Permettre aux utilisateurs de changer leur email. 1 pour oui, 0 pour non', NULL, NULL, NULL),
+(38, 'fr_FR', 'Demander aux clients de confirmer leur email. 1 pour oui, 0 pour non', NULL, NULL, NULL),
```
Comment (lunika) [inline setup/insert.sql]: don't put a comma at the end of this line.

**Comment by lunika** [comment]
It seems that you have the same modification in PR #1132 

Did you start your development from your branch ```customer-change-email``` instead of ```master``` ?

Maybe you can close the older and keep this one, but verify first if the modification are exactly the same.

**Comment by gillesbourgeat** [comment]
ok, the PR are marged.
PR #1132 is closed

**Comment by lunika** [comment]
Thanks

As you can see here, you break the API with your modification : https://travis-ci.org/thelia/thelia/builds/50196524

**Pull request merged**